addons/point_of_sale/models/pos_order.py

```diff
@@ -714,6 +714,7 @@ def action_pos_order_invoice(self):
 
             inv = invoice._convert_to_write({name: invoice[name] for name in invoice._cache})
             new_invoice = Invoice.with_context(local_context).sudo().create(inv)
+            new_invoice.team_id = self.config_id.crm_team_id
```

Comment:
Can't we update `team_id` in `inv` vals ? (or here https://github.com/odoo/odoo/blob/5203b6482fcc7ae5c265a55c1b79e9c331d9d4a0/addons/point_of_sale/models/pos_order.py#L189)

Reply:
@rhe-odoo the field 'crm_team_id' does not exist in 'point_of_sale' module. It is introduced in pos_sale. So you have to override the function '_prepare_invoice' in module pos_sale to add the 'team_id' field on invoice. More over, the invoices don't contain the field 'team_id' if sale is not installed. So it could not work without pos_sale.